examples/tools/android_native_run/readme.md

```diff
@@ -0,0 +1,207 @@
+# Support for Android native libraries to QBDL
+
+
+The goal of this project was to launch a Java Virtual Machine which is able to support the unknown android symbols. 
+For our application, we are using some code from android studio. The c++ program will then start the JVM in which
+the Java code from android studio will be able to compil thanks to the imported libraries.
+
+
+## How to use it :
+
+### Java classes
+
+To compile the code, you will need to know the paths to the Java classes. Then create a .txt document in which you can gather all these paths. To do so, you can use this command :
+
+
+`find ~/.gradle/caches -name "*-runtime.jar" > dependencies.txt`
+
+It will find all the java dependencies ending with "runtime.jar" which are necessary for the code to be well executed. Other files are required although. You have to include these two paths into the same file :
+
+> The Android runtime, which you can normally find in your home directory followed by :
+`/Android/Sdk/platforms/android-32/android.jar`
```

Comment:
Using a `>` here makes things look a bit weird when rendering the markdown, you can probably use a list here:
```suggestion
  - The Android runtime, which you can normally find in your android's SDK installation followed by `/platforms/android-32/android.jar`.
  - The directory where the Java classes using the native libraries are compiled; on my setup it was here:
`~/dev/example-android-native-app/app/build/intermediates/javac/debug/classes`
```

Reply:
Done